pressbooks/pressbooks-aldine
# feat: new front page catalog

This PR address #365 

How to test:

1. Navigate to the customizer and enable the front end catalog
2. Select your 4 books of the catalog you should preview after selecting each of them
3. Review if your selection is previewed in the book customizer
4. Try to use the same book in two selects (it should not be possible)
5. Enjoy your new simplified front page
6. The old catalog should be working as before

![image](https://user-images.githubusercontent.com/4997549/199848297-8f64b862-14de-4373-a048-fd149520b2d9.png)
